test/templates/dialog/component.njk

```diff
@@ -0,0 +1,35 @@
+{% import "../forms.njk" as forms %}
+{% from "../button/component.njk" import button %}
+{% from "../icon/component.njk" import icon %}
+<!--
+dialog:
+    properties={},
+-->
+{% macro dialog(properties={}) %}
+<div class="fd-dialog--{{properties.size}}" role="dialog">
+  <div class="fd-dialog__content" role="document">
+      <header class="{{properties.class}}">
```

Comment:
This is also a modifier but for an element. This could be passed as `modifier.header` and the same modifier filter can be used to apply the class.
```
<header class="fd-dialog__header{{ modifier.header | modifier('dialog__header') }}">
```